src/main/steps/c100-rebuild/applicant/util.ts

```diff
@@ -0,0 +1,10 @@
+import { C100Applicant, ChildrenDetails } from '../../../app/case/definition';
+
+export const getApplicantDetails = (applicants: C100Applicant[] | [], applicantId: string): C100Applicant | undefined =>
```

Comment:
Can we refactor applicant & respondent utils to make sure to reuse methods please.